src/MinecraftSkins.php

```diff
@@ -50,6 +58,14 @@ public static function skin($rawSkin, $scale = 1, $vertRot = 0, $horRot = 0, $he
         imagecopyresampled($canvas, $head, 4 * $scale, 0 * $scale, 0, 0
                 , self::HEAD_SIZE * $scale, self::HEAD_SIZE * $scale
                 , self::HEAD_SIZE * $scale, self::HEAD_SIZE * $scale);
+        
+		// helmet layer
+		if ($helmet == true){
```

Comment:
Please use spaces instead of tabs, for this project. Editors and IDEs can be changed to do that automatically on tab.

Reply:
Replace all tabs in new branch